src/main/scala/is/hail/expr/AST.scala

```diff
@@ -517,6 +527,89 @@ case class Apply(posn: Position, fn: String, args: Array[AST]) extends AST(posn,
 
         `type` = tNew
 
+      case "ungroup" =>
+        if (args.length != 3)
+          parseError(
+            s"""invalid arguments for method `$fn'
+               |  Usage: $fn(Struct, identifier, mangle)
```

Comment:
I started to do that, but it made the code substantially more complicated. I figured since this is for 0.2, the call to the expression language will be from a Python function so we can handle the default argument in Python.